gramps/gen/utils/id.py

```diff
@@ -46,11 +46,19 @@
 # create_id
 #
 #-------------------------------------------------------------------------
-rand = random.Random(time.time())
+_rand = random.Random(time.time())
+_det_id = False
+
 
 def create_id():
-    return "%08x%08x" % (int(time.time()*10000),
-                         rand.randint(0, sys.maxsize))
+    global _rand
+    if _det_id is True:
```

Comment:
Better to just leave out "is True", unless you are specifically testing between other values of True.